c/tskit/convert.c

```diff
@@ -46,6 +46,7 @@ typedef struct {
     size_t precision;
     tsk_flags_t options;
     char *newick;
+    tsk_id_t *traversal_stack;
```

Comment:
Is there any reason to be concerned about thread safety for the newick converter?

Reply:
No, but we want to get rid of the stacks that are in the ``tsk_tree_t`` structure, as that was a Bad Idea in the first place.

Reply:
Looks good then!